recipes/default.rb

```diff
@@ -21,7 +21,7 @@
 
 # Start+Enable the lvmetad service on RHEL7, it is required by default
 # but not automatically started
-if node['platform_family'] == 'rhel' && node['platform_version'].to_i >= 7
+if node['platform_family'] == 'rhel' && node[platform] != 'amazon' && node['platform_version'].to_i >= 7
```

Comment:
personally i even think the `if node['platform_family'] == 'rhel' && node['platform_version'].to_i >= 7` should be removed. Because it's not a matter of linux distro that defines whether `lvm2-lvmetad` is required for lvm to work. It's the setting of the lvm configuration 

```
[ec2-user@ip-172-31-28-89 ~]$ sudo /sbin/lvm dumpconfig
config {
...
}
dmeventd {
...
}
activation {
...
}
global {
...
	use_lvmetad=0
}
shell {
	history_size=100
}
backup {
...
}
log {
...
allocation {
...
}
devices {
...
}
```

that defines whether `lvm2-lvmetad` is required. It's just that different linux distro decides to tune these configuration differently from each other (after all, linux distro belongs to the same family).